particles/__init__.py

```diff
@@ -145,7 +142,7 @@ def regime(self):
     @property
     def in_equilibrium(self):
         """ Simple check for equilibrium """
-        return self.T > self.decoupling_temperature
+        return (self.T > self.decoupling_temperature) * self.thermal
```

Comment:
Can you say in words what this method now does? "Particle is in equilibrium if it is thermal and T>decoupling temperature"?

Then it should look like this:

```return self.thermal and self.T > self.decoupling_temperature ```

Reply:
The parameter thermal is used to make a distinction between particles that start with an equilibrium distribution (thermal = True) and particles that don't (thermal = False). So now the code can deal with particles coming from HNL decay that weren't there before in the plasma. We calculate Boltzmann equation for (most of) these particles and for this they must be out of equilibrium to pass the check in four_particle/__init__.py.

Reply:
So that's where you are wrong. We don't have particles that "weren't there before". All they are present in equilibrium quantities even if they happen to be tiny. There are always e^(-105/T) muons in the plasma.

This reminds me that I didn't see the part of the code that keeps muons having a temperature. Have you done this yet?

Reply:
They will be in kinetic equilibrium, but not in chemical equilibrium. Those latter reactions are out of equilibrium. 
I do need to add the temperature part. Even though some particles will not have the same temperature as plasma (neutral pions for example).